src/components/molecules/NavBar/NavBar.scss

```diff
@@ -576,3 +584,15 @@ $border-radius: 28px;
     }
   }
 }
+
+@media (max-width: 600px) {
+  .schedule-text {
+    display: none;
+  }
+}
+
+@media (max-width: 900px) {
+  .nav-search-links {
+    display: none;
+  }
+}
```

Comment:
[NB] Do we have helpers for handling this in a centralised way yet? (similar to our `z()` helper) I know it's been discussed/raised in PRs before, but unsure if we've done it yet.

Reply:
Seems we've started to. Would be good to have more added there: https://github.com/sparkletown/sparkle/blob/staging/src/scss/constants.scss#L52

See https://github.com/sparkletown/internal-sparkle-issues/issues/4#issuecomment-796371907 / related